pandas/core/frame.py

```diff
@@ -2212,6 +2212,83 @@ def insert(self, loc, column, value, allow_duplicates=False):
         self._data.insert(
             loc, column, value, allow_duplicates=allow_duplicates)
 
+    def assign(self, **kwargs):
+        """
+        Assign new columns to a DataFrame.
+
+        Parameters
+        ----------
+        **kwargs : keyword, value pairs
+            keywords are the column names, and values are either
+            inserted into the DataFrame, or called on the DataFrame
+            and inserted if the value is callable.
+
+        Returns
+        -------
+        df : DataFrame
+            A new DataFrame with the new columns in addition to
+            all the existing columns.
+
+        Notes
+        -----
+        Since **kwargs is a dictionary, the order of your
```

Comment:
The `**` in `**kwargs` will have to be escaped (because `*` are also used for bold, and this will give a warning for 'Inline emphasis start-string without end-string.')

I think just `\**kwargs` will do it